### ISSUE TITLE
Extension breaks Gnome 3.36 desktop search!?

### ISSUE THREAD

**Issue by johnnydvc** [Maestroschan/appfolders-manager-gnome-extension#102]
Hi.
Me and some other Arch users have a broken gnome shell search thanks to this extension.
See https://www.reddit.com/r/gnome/comments/fluzzh/gnome_shell_336_desktop_search_brocken/.
The only way for now is to disable it.
Thanks

**Comment by maoschanz** [comment]
>The only way for now is to disable it.

So disable it?

I don't provide any support for 3.36, i intentionally didn't add "3.36" to the manifest file, i don't even understand why people use this extension with a GNOME Shell version *where it makes no sense at all since the feature natively exists* since 3.34. If your distro installs it by default, i'm not in charge of their choice

**Comment by johnnydvc** [comment]
Yeah, I just realised that it's included natively in 3.36. Sorry if it sounded harsh. I really like the extension and I'm happy that the folks from gnome think so too.

**Comment by OiYouYeahYou** [comment]
Erm, I was having issues as well and am on `3.36`. But I wasn't told or warned this gnome shell version isn't supported when installing via the website and chrome extension 😥

Is there a way for the extension to check the shell version and just refuse to work, preventing further problems?

Anyway, thank you for the utility this has provided previously
